### PR TITLE
Indy reorganization

### PR DESCRIPTION
**Pull request by andrewwhitehead**
- Move current issuer, holder, verifier support under indy subpackage, renaming Base* to Indy*
- Update test package dependencies
- Add support for `askar`, `indy_credx`, `indy_vdr` test tags and automatic test skipping

### CODE REVIEW

File: aries_cloudagent/core/tests/test_conductor.py [hunk 1]
```diff
@@ -506,11 +506,11 @@ async def test_admin_startx(self):
         await wallet.create_public_did()
 
         with async_mock.patch.object(
-            admin, "start", async_mock.CoroutineMock()
+            admin, "start", auto_spec=True
```
Comment (sklump) [inline aries_cloudagent/core/tests/test_conductor.py:509]: autospec here, I think? (no underscore)

File: aries_cloudagent/core/tests/test_conductor.py [hunk 2]
```diff
@@ -506,11 +506,11 @@ async def test_admin_startx(self):
...
         ) as admin_start, async_mock.patch.object(
             admin, "stop", autospec=True
         ) as admin_stop, async_mock.patch.object(
-            test_module, "ConnectionManager"
+            test_module, "ConnectionManager", auto_spec=True
```
Comment (sklump) [inline aries_cloudagent/core/tests/test_conductor.py:513]: autospec

**Review by sklump — DISMISSED** [review]

**Comment by ianco** [comment]
I get errors running the Faber demo with these updates:

```
#3/4 Create a new schema/cred def on the ledger
Faber      | 2020-11-23 20:54:11,740 aries_cloudagent.core.dispatcher ERROR Handler error: schemas_send_schema
Faber      | Traceback (most recent call last):
Faber      |   File "/home/indy/aries_cloudagent/messaging/schemas/routes.py", line 131, in schemas_send_schema
Faber      |     issuer: IndyIssuer = await context.inject(IndyIssuer)
Faber      |   File "/home/indy/aries_cloudagent/config/injection_context.py", line 126, in inject
Faber      |     return await self.injector.inject(base_cls, settings, required=required)
Faber      |   File "/home/indy/aries_cloudagent/config/injector.py", line 77, in inject
Faber      |     result = await provider.provide(ext_settings, self)
Faber      |   File "/home/indy/aries_cloudagent/config/provider.py", line 106, in provide
Faber      |     instance = await self._provider.provide(config, injector)
Faber      |   File "/home/indy/aries_cloudagent/config/provider.py", line 52, in provide
Faber      |     instance_cls = ClassLoader.load_class(instance_cls)
Faber      |   File "/home/indy/aries_cloudagent/utils/classloader.py", line 109, in load_class
Faber      |     raise ClassNotFoundError(f"Module '{mod_path}' not found")
Faber      | aries_cloudagent.utils.classloader.ClassNotFoundError: Module 'aries_cloudagent.issuer.indy.sdk' not found
Faber      | 2020-11-23 20:54:11,741 aries_cloudagent.admin.server ERROR Handler error with exception: Module 'aries_cloudagent.issuer.indy.sdk' not found

```

**Review by ianco — CHANGES_REQUESTED** [review]
Error running Faber demo

**Comment by andrewwhitehead** [comment]
Thanks @ianco, should be fixed now. I'm surprised that wasn't caught by the tests.

**Review by ianco — APPROVED** [review]

**Pull request merged**